sdk/identity/tests.yml

```diff
@@ -24,9 +24,11 @@ extends:
       - pwsh: |
           [System.Convert]::FromBase64String($env:PFX_CONTENTS) | Set-Content -Path $(Agent.TempDirectory)/test.pfx -AsByteStream
           Set-Content -Path $(Agent.TempDirectory)/test.pem -Value $env:PEM_CONTENTS
+          [System.Convert]::FromBase64String($env:SNI_CONTENTS) | Set-Content -Path $(Agent.TempDirectory)/testsni.pfx -AsByteStream
```

Comment:
Don't we have access to `IDENTITY_SP_CERT_SNI` here already?

Reply:
I'm not sure I understand the question. `net-identity-spcert-pfx` (saved to `test.pfx`) and `net-identity-spcert-sni` (saved to `testsni.pfx`) are two different certificates. I've set it up so they can both authenticate the application specified by `net-identity-sp-clientid`, but the certificate `net-identity-spcert-sni` isn't specifically registered with the application but it's subject name and issuer are. It is used to test Subject Name / Issuer authentication.